### PR TITLE
Bump proforma & add metadata

### PR DESCRIPTION
**Pull request by kkoehn**
- Update proforma gem
- add meta_data fields to test and task
- update import/export flow to support meta_data

### CODE REVIEW

**Review by MrSerth — COMMENTED** [review]
Thank you for taking the time to include the latest proforma gem and include the updated metadata format. I like storing the metadata as JSONB to the database which will also allow us to query the data directly, right? That's great news ☺️

I just have a small remark regarding the new serializer (which I didn't know would exist in Rails that way, but found it interesting to read more about that!). So thanks for pointing that out 😁

File: app/serializers/hash_as_jsonb_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class HashAsJsonbSerializer
+  def self.dump(hash)
+    hash
+  end
+
+  def self.load(hash)
+    # (hash.is_a?(Hash) ? hash : {}).with_indifferent_access
```
[the remark this answers is not in the document]
Comment (kkoehn) [reply app/serializers/hash_as_jsonb_serializer.rb:9]: The problem with `with_indifferent_access` was, that the hash then has the class `HashWithIndifferentAccess` and that would currently break the meta_data serialization in gem

File: app/serializers/hash_as_jsonb_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class HashAsJsonbSerializer
```
[the remark this answers is not in the document]
Comment (kkoehn) [reply app/serializers/hash_as_jsonb_serializer.rb:3]: good idea

File: app/serializers/hash_as_jsonb_serializer.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (MrSerth) [reply app/serializers/hash_as_jsonb_serializer.rb]: Ah, good to know, thanks!

**Review by MrSerth — APPROVED** [review]
Well done, thank you! 🚀

**Pull request merged**